packages/components/src/DataList/components/DataListFilters/DataListFilters.tsx

```diff
@@ -1,9 +1,6 @@
 import React, { ReactElement } from "react";
-import classNames from "classnames";
-import { useInView } from "@jobber/hooks/useInView";
-import styles from "./DataListFilters.css";
-import { CONTAINER_TEST_ID } from "./DataListFilter.const";
 import { DataListSort } from "./components/DataListSort";
+import { DataListOverflowFade } from "./components/DataListOverflowFade/DataListOverflowFade";
```

Comment:
consistency wise, add an index.ts on this folder and import it via

```suggestion
import { DataListOverflowFade } from "./components/DataListOverflowFade";
```